wgpu-native/Cargo.toml

```diff
@@ -16,9 +16,11 @@ license = "MPL-2.0"
 crate-type = ["lib", "cdylib", "staticlib"]
 
 [features]
-default = []
+default = ["x11"]
```

Comment:
This will make it so `gfx-backend-vulkan` is always enabled as a feature (and it's a feature on Apple platforms)